src/tests/tribol_mfem_common_plane.cpp

```diff
@@ -238,12 +241,13 @@ class MfemCommonPlaneTest : public testing::TestWithParam<int> {
 
 TEST_P(MfemCommonPlaneTest, mass_matrix_transfer)
 {
-  EXPECT_LT(std::abs(max_disp_ - 0.0000033982222612802126), 1.0e-8);
+  EXPECT_LT(std::abs(max_disp_ - 0.013637427890739103), 1.0e-8);
```

Comment:
Just so I know, why did the value change so much? Different penalty?

Reply:
Yeah, the new penalty is much higher so there's a lot more separation at the end of the sim.